mteb/benchmarks/benchmarks.py

```diff
@@ -62,8 +62,61 @@ def load_results(
         return base_results.select_tasks(self.tasks)
 
 
-MTEB_MAIN_EN = Benchmark(
```

Comment:
This var is referenced in 2 other files: `docs/adding_a_model.md` and `mteb/__init__.py`. We might want to reuse that for either the MMTEB one or the classic one.

Reply:
Fixed